### DISCUSSION TITLE
How to access lat and lon at a given time?

### DISCUSSION THREAD

**Discussion by Manzi-Fabrice** [OpenDrift/opendrift#1426] [Q&A]
I am seeding 100 elements and saving data as .nc and then I convert the result to .csv however I am only seeing 10 elements is this the configuration that limits the number of ouput or some it's a bug in my code>

**Comment by knutfrode** [comment]
Hi, this is hard to tell from this short description, but appears to be a bog in your conversion to csv.
But if using `loglevel=0` (debug), you should see what happens to your elements.

Note that you may use the TrajAn software for analysis/post-processing
https://opendrift.github.io/trajan/

This does not have proper methods for animationss yet, but it will come.
